build.gradle

```diff
@@ -51,6 +51,20 @@ tasks.named(sourceSets.main.processResourcesTaskName, Copy) {
     from(actionsTemplateZip)
 }
 
+tasks.named('jar', Jar) {
+    final ByteArrayOutputStream bos = new ByteArrayOutputStream()
+    project.exec {
+        commandLine('git', 'rev-parse', 'HEAD')
+        standardOutput(bos)
+    }.rethrowFailure().assertNormalExitValue()
+
+    final gitCommit = new String(bos.toByteArray())
+    // Store the git commit as the Implementation-Title of the tasks package
+    manifest.attributes([
+            'Implementation-Title': gitCommit
```

Comment:
I'm a bit... _meh_ on storing information in the JAR manifest. In part because of future-proofing concerns, how Java 9 modules don't have this information. What other alternatives do we have for this?

Also, why the implementation _title_? Why not the implementation _version_?

Reply:
Reading non-standard information from the manifest is not very straight-forward. I will add a TODO to re-evaluate this in the future and in the meantime switch it to the version.